Documentation/Getting-Started/Prerequisites/prerequisites.md

```diff
@@ -9,6 +9,10 @@ and Rook is granted the required privileges (see below for more information).
 
 Kubernetes **v1.17** or higher is supported for the Ceph operator.
 
+## CPU Architecture  
+
+We only support `amd64 / x86_64` and `arm64` architecture.  
```

Comment:
How about this wording? "Support" is more of a downstream or product term.
```suggestion
Architectures released are `amd64 / x86_64` and `arm64`.
```

Reply:
@travisn sure, sounds good.